liblouis/lou_translateString.c

```diff
@@ -2839,7 +2840,8 @@ resolveEmphasisResets(EmphasisInfo *buffer, const EmphasisClass class,
 						if (wordBuffer[i] & WORD_RESET ||
 								!checkAttr(input->chars[i], CTC_Letter, 0, table)) {
 							if (!checkAttr(input->chars[i], CTC_Letter, 0, table)) {
-								if (checkAttr(input->chars[i], CTC_CapsMode, 0, table)) {
```

Comment:
I understand that you added the emph check below, but shouldn't there be a separate one for caps?

Reply:
I think this is OK. @bertfrees makes the `resolveEmphasisResets` more generic by adding a `emphModeCharsAttr` argument. He then calls the function twice, once with `CTC_CapsMode` and once with `CTC_EmphMode` (if the table uses emph mode.